test/extern/kbrady/rename_extern_record.chpl

```diff
@@ -0,0 +1,15 @@
+extern "struct stat" record chpl_stat {
+  var st_size: off_t;
+}
+
+proc getFileSize(path:c_string) : int {
+  extern proc stat(x: c_string, ref chpl_stat): c_int;
+  var buf: chpl_stat;
+
+  if (stat(path, buf) == 0) {
+    return buf.st_size;
+  }
+  return -1;
+}
+
+writeln(getFileSize("./rename_extern_record.good"));
```

Comment:
Checking against the .good file's size seems like asking for trouble to me.  If someone wants this test to print out more information later, they've have to redetermine the file size, too.